index_settings_test.go

```diff
@@ -378,6 +380,47 @@ func TestIndex_GetSortableAttributes(t *testing.T) {
 	}
 }
 
+func TestIndex_GetTypoTolerance(t *testing.T) {
+	type args struct {
+		UID    string
+		client *Client
+	}
+	tests := []struct {
+		name     string
+		args     args
+		wantResp *TypoTolerance
```

Comment:
What about just `response` or `expectation`?

Reply:
Same, as it is autogenerated I didn't change the name of the variable